Core/DISET/private/Transports/SSL/SocketInfoFactory.py

```diff
@@ -121,8 +132,9 @@ def getSocket( self, hostAddress, **kwargs ):
     retVal = Network.getIPsForHostName( hostName )
     if not retVal[ 'OK' ]:
       return S_ERROR( "Could not resolve %s: %s" % ( hostName, retVal[ 'Message' ] ) )
-    ipList = List.randomize( retVal[ 'Value' ] )
-    for i in range( 3 ):
+    ipList = retVal[ 'Value' ] #In that case the first ip always  the correct one.  
+    
+    for _ in range( 1 ): #TODO: this retry can be reduced. 
```

Comment:
I prefer to keep it, because it maybe need to be increased to 2. It will easy to me to found out what needs to extend.